bottom/client.py

```diff
@@ -1,14 +1,20 @@
 import asyncio
 import collections
 import functools
+from typing import Any, Callable, DefaultDict, List, Optional  # noqa
 from bottom.protocol import Protocol
 from bottom.pack import pack_command
 
 
 class Client:
-    protocol = None
+    protocol = None  # type: Optional[Protocol]
 
-    def __init__(self, host, port, *, encoding="UTF-8", ssl=True, loop=None):
+    _handlers = None  # type: DefaultDict[str, List[Callable]]
+    _events = None  # type: DefaultDict[str, asyncio.Event]
+
+    def __init__(self, host: str, port: int, *,
+                 encoding: str = "UTF-8", ssl: bool = True,
```

Comment:
Does PEP 484 recommend spaces/no spaces around the equals sign for default values?  Without types I've used `some_param=None` so I'd expect this to be `some_param: Optional[Thing]=None`.